src/MultiSelectAutocomplete/MultiSelectAutocomplete.scss

```
@@ -0,0 +1,122 @@
+$govuk-font-family: 'Roboto', arial, sans-serif;
+
+@import "node_modules/govuk-frontend/govuk/_base";
+@import "node_modules/govuk-frontend/govuk/components/input";
+@import "node_modules/govuk-frontend/govuk/components/hint";
+
+.hods-multi-select-autocomplete__outer-wrapper {
+  
+  .hods-multi-select-autocomplete {
+  
+    .error__control,
+    &__control {
+      width: 100%;
+      border: 2px solid govuk-colour("black");
+      border-radius: 0;
+      padding: 0;
+      font-size: 19px;
+      box-sizing: border-box;
+      transition: none;
+
+      &--is-focused {
+        outline: 3px solid govuk-colour("yellow") !important;
+        outline-offset: 0;
+        box-shadow: inset 0 0 0 2px;
+      }
+
+      &--is-disabled {
+        background-color: #efefef4d;
+      }
+
+      &:hover {
+        border: 2px solid $govuk-input-border-colour;
+      }
+    }
+    
+    .error__value-container,
+    &__value-container {
+      padding: 5px;
```

Seeing as you're using `rems` in the lines below, this should be in the same units:
```suggestion
      padding: 0.3125rem;
```
(5 / 16 = 0.3125)